metrics-core/src/main/java/com/codahale/metrics/ScheduledReporter.java

```diff
@@ -145,6 +145,26 @@ public void run() {
     }
 
     /**
+     * Starts the reporter polling at the given period.
```

Comment:
Gnnhhh ... my IDE is configured to replace ".*" imports with the explicit ones - but I did NOT want to push those changes, as they were not necessary for the fix.